src/pages/About.jsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from "react-router-dom";
+
+export default function About() {
+  return (
+    <div className="about-page-container">
+      <img src={"/assets/about-hero.png"} className="about-hero-image" />
```

Comment:
You don't need to wrap the string in the curly braces, it's unnecessary to do so. You should remove them.